fuzz/type_info_adt_set.c

```diff
@@ -45,10 +45,16 @@ alloc_scen(struct theft *t, void *type_env, void **output)
 		switch (op->t) {
 		case SET_OP_ADD:
 			op->u.add.item = theft_random_bits(t, env->item_bits);
+			if (op->u.add.item == 0) {
+				return THEFT_ALLOC_SKIP;
```

Comment:
Returning `SKIP` here will leak `res`. It's probably not a big deal, except for extremely long fuzzing runs, but you could call `free(res);` first.

Reply:
Thanks!